modules/apps/company/company-test/src/testIntegration/java/com/liferay/company/service/test/CompanyLocalServiceDBPartitionTest.java

```diff
@@ -819,6 +901,25 @@ connection, getPartitionName(copiedCompanyId)),
 		}
 	}
 
+	private void _assertRepositoryClassDefinitionsCacheCleanup(long companyId) {
```

Comment:
What I meant is to check all maps an cleared, including the className and counter